include/cuda/std/barrier

```diff
@@ -714,6 +742,15 @@ void inline __memcpy_async_sm_dispatch(
         std::size_t __size, _Sync & __sync, async_contract_fulfillment & __is_async) {
     // Broken out of __memcpy_async to avoid nesting dispatches
     NV_DISPATCH_TARGET(
+        NV_PROVIDES_SM_90,
+            if (__is_bulk_enabled() && __isShared(&__sync)) {
```

Comment:
What is the purpose of `__is_bulk_enabled()` if it always returns `true`? 

```suggestion
            if (__is_bulk_enabled() && __isShared(&__sync)) {
```

Reply:
Just a performance knob. There's more code that will follow up and will use this